### ISSUE TITLE
How do you get bootstrap servers for a serverless MSK cluster?

### ISSUE THREAD

**Issue by naddeoa** [pulumi/pulumi-aws#2279]
### What happened?

I'm trying to convert my standard MSK cluster into a serverless one. Previously, I would get a hold of the bootstrap servers for the kafka client like this

```typescript
const kafkaCluster = new aws.msk.Cluster(serviceName, {
    // ...
});
export const zookeeperConnectString = kafkaCluster.zookeeperConnectString;
export const bootstrapBrokersTls = kafkaCluster.bootstrapBrokersTls;
export const bootstrapBrokers = kafkaCluster.bootstrapBrokers;
```

The `aws.msk.ServerlessCluster` type doesn't have those fields and the [serverless cluster](https://www.pulumi.com/registry/packages/aws/api-docs/msk/serverlesscluster/) docs don't mention anyway of getting the bootstrap server. I tried to call `aws.msk.getBrokerNodes` to see if it was in there by that function apparently isn't valid for serverless msk.

I can see my bootstrap server in the AWS console for the serverless cluster that pulumi makes, and I can look it up via the AWS cli with `aws kafka get-bootstrap-brokers`, but I can't find anything in pulumi that returns that info. 

### Steps to reproduce

n/a

### Expected Behavior

I expected the `aws.msk.ServerlessCluster` type to have the same properties as `aws.mask.Cluster`, especially in regards to `aws.mask.Cluster.bootstrapBrokers`.

### Actual Behavior

No apparent way to get the bootstrap servers.

### Output of `pulumi about`

CLI          
Version      3.49.0
Go Version   go1.19.3
Go Compiler  gc

Host     
OS       ubuntu
Version  22.04
Arch     x86_64

Backend        
Name           pulumi.com
URL            https://app.pulumi.com/whyinfra
User           whyinfra
Organizations  whyinfra

Pulumi locates its logs in /tmp by default

### Contributing

Vote on this issue by adding a 👍 reaction. 
To contribute a fix for this issue, leave a comment (and link to your pull request, if you've opened one already).

**Comment by danielrbradley** [comment]
Hi @naddeoa if I understand your issue correctly, you're looking to add an output for the bootstrap server url. Is that correct?

It looks like we're blocked by the upstream implementation which uses https://docs.aws.amazon.com/cli/latest/reference/kafka/describe-cluster-v2.html which does not yet contain this information. Once that upstream issue is resolved, this should be able to be made available. If it's able to be accessed via the CLI, a temporary workaround could be to use the [Command Provider](https://www.pulumi.com/registry/packages/command/) to run the required CLI command.

**Comment by naddeoa** [comment]
Yes, that's right. Ok, thanks for the work around. I'll probably delay until the other issue is resolved.

**Comment by nivinjoseph** [comment]
Wow, this has still not been addressed?

**Comment by grahamdaley** [comment]
Two years later and still not addressed?
